Formula/certigo.rb

```diff
@@ -0,0 +1,18 @@
+class Certigo < Formula
+  desc "Utility to examine and validate certificates in a variety of formats"
+  homepage "https://github.com/square/certigo"
+  url "https://github.com/square/certigo.git"
+  version "1.1.0"
+  sha256 "1ca8ee1130d57fb70b1c21cc9311d8b036b86d456eb88888d603e88fbb1d056a"
+
+  depends_on "go" => :build
+
+  def install
+    system "./build"
+    bin.install "bin/certigo"
+  end
+
+  test do
+    system "certigo", "help"
```

Comment:
Please use `"#{bin}/certigo"`. Also, can’t we write a test that does something more substantial?